packages/cache-symfony/tests/Jellyfish/CacheSymfony/CacheSymfonyServiceProviderTest.php

```diff
@@ -43,9 +43,7 @@ protected function _before(): void
 
         $self = $this;
 
-        $this->container->offsetSet(ConfigConstants::FACADE, static function () use ($self) {
-            return $self->configFacadeMock;
-        });
+        $this->container->offsetSet(ConfigConstants::FACADE, static fn() => $self->configFacadeMock);
```

Comment:
`$this` assignment can be removed, arrow function don't have their own this context.